tests/test_queue.py

```diff
@@ -176,7 +176,7 @@ async def test_retry(self) -> None:
 
     async def test_retry_delay(self) -> None:
         # Let's first verify how things work without a retry delay
-        worker = Worker(self.queue, functions=functions, dequeue_timeout=0.01)
+        worker = Worker(self.queue, functions=functions, dequeue_timeout=0.1)
```

Comment:
The low timeout was causing this test to be flaky since the Postgres queue isn't as fast as the Redis one.